pkg/yamltemplate/filetests/ytt-library/assert-equals-err.yml

```diff
@@ -0,0 +1,11 @@
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:json", "json")
+
+test_equals: #@ assert.equals("foo", "bar")
+
++++
+
+ERR: 
+- assert.equals: expected foo, but found bar
```

Comment:
This error could use a little more description as to why we were expecting 'foo'. Maybe a message with the the words not equal in it would help. For example: 'unequal: bar not equal to foo'.

Reply:
changed to: `unequal: bar is not equal to the expected value foo`